app/api/views/questions.py

```diff
@@ -0,0 +1,81 @@
+from flask import request, jsonify
+from flask_login import login_required, current_user
+from app.api import api
+from app.api.business import questions_business
+from app.api.helpers import (
+    abort,
+    forbidden,
+    not_found,
+    get_email_domain,
+    role_required,
+    permissions_required
+)
+from app.api.business.errors import (
+    NotFoundError,
+    ValidationError,
+    UnauthorisedError
+)
+from ...utils import get_json_from_request
+
+
+@api.route('/brief/<int:brief_id>/questions', methods=['GET'])
+@login_required
+@role_required('buyer')
+def get_questions(brief_id):
+    result = None
+    try:
+        result = questions_business.get_questions(current_user, brief_id)
+    except NotFoundError as nfe:
+        not_found(nfe.message)
+    except UnauthorisedError as ue:
+        return forbidden(ue.message)
```

Comment:
With these views, do we need to catch any other `Exception` and report to Rollbar?

Reply:
done